scene/theme/default_theme.cpp

```diff
@@ -816,6 +817,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_constant("children_hl_line_width", "Tree", 1);
 	theme->set_constant("parent_hl_line_margin", "Tree", 0);
 	theme->set_constant("draw_guides", "Tree", 1);
+	theme->set_constant("draw_stripes", "Tree", 1);
```

Comment:
```suggestion
	theme->set_constant("draw_stripes", "Tree", 0);
```
As pointed out in the proposal this should be off by default